apps/web/package.json

```diff
@@ -104,6 +104,7 @@
     "react-use-intercom": "1.4.0",
     "react-virtualized-auto-sizer": "^1.0.6",
     "react-window": "^1.8.6",
+    "rrule": "^2.6.9",
```

Comment:
`rrule` helps us take control of the different configurations around recurring events using the official standard, being able to extend functionality easily in the future. Here's the health of it FYI
<img width="458" alt="rrule_-_npm_Package_Health_Analysis___Snyk" src="https://user-images.githubusercontent.com/467258/164992966-6dfd45fc-7264-4d3a-9116-eefdd047d76d.png">
:

Reply:
+2 on Rule - awesome package